manifest-build-tools/application/make_debian_packages.py

```diff
@@ -217,10 +218,22 @@ def write_downstream_parameter_file(build_directory, is_official_release, parame
         else:
             raise RuntimeError("Version of {0} is None. Maybe the repository doesn't contain debian directory ".format(rackhd_repo_dir))
 
+        manifest = Manifest(manifest_file)
+        rackhd_commit = ''
```

Comment:
the word ```rackhd_commit``` will brings many meanings. 
here it means git repo "RackHD/RackHD" 's commit , which the downstream jobs should checkout to.
so a comment appended would be better.